### PR TITLE
Issue 99 fix #99

### LINKED ISSUE #99: Intégrer outil choix CCN
Proposer l'outil lorsque l'on recherche sa convention collective

### PR DESCRIPTION
**Pull request by revolunet**
ajout datasets + indexation CCNs

### CODE REVIEW

File: packages/code-du-travail-frontend/src/sources.js [hunk 1]
```diff
@@ -1,24 +0,0 @@
-// mapping elastic search source type -> route name
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/sources.js]: Why did you remove it ?

Comment (revolunet) [reply packages/code-du-travail-frontend/src/sources.js]: argh PR splitting...

Comment (lionelB) [reply packages/code-du-travail-frontend/src/sources.js]: I see that in the PR "Unify Content Route" ;)

Comment (revolunet) [reply packages/code-du-travail-frontend/src/sources.js]: ok je remix :)

**Review by lionelB — APPROVED** [review]
